src/db.rs

```diff
@@ -49,11 +49,16 @@ pub struct KbsDb {
 
 impl KbsDb {
     pub async fn new() -> Result<Self> {
-        let db_type = env::var("KBS_DB_TYPE").expect("KBS_DB_TYPE not set");
-        let host_name = env::var("KBS_DB_HOST").expect("KBS_DB_HOST not set");
-        let user_name = env::var("KBS_DB_USER").expect("KBS_DB_USER not set.");
-        let db_pw = env::var("KBS_DB_PW").expect("KBS_DB_PW not set.");
-        let data_base = env::var("KBS_DB").expect("KBS_DB not set");
+        let db_type = env::var("KBS_DB_TYPE")?;
```

Comment:
Will we lose the descriptive error messages here? It might be nice to add a `map_err` thing here to set the error message to something descriptive.

Reply:
These have been fixed to use a map_err which will yield a more informative error message.